gathertown/sendbird-uikit-react
# Extend SBUEventHandlers to take a request onFailed event

## Description

Expose a generic event handler for request failures.

## Test Plan

1. Integrate with Gather Browser
2. Add an event handler for request onFailed in ChatProvider
```
const onRequestFailed = (error: SendBirdError | Error, context: Object) => {
   console.log({error, context}
}

 eventHandlers={{ connection: { onFailed: onConnectionFailed }, request: { onFailed: onRequestFailed } }}
````
3. Open chat
4. Disconnect from the internet
5. Open chat channel
6. Observe an error is console.log'd out
